cpp/log/etcd_consistent_store_test.cc

```diff
@@ -289,8 +289,8 @@ TEST_F(EtcdConsistentStoreTest, TestSetServingSTHWontOverwriteWithOlder) {
   EXPECT_EQ(util::error::OUT_OF_RANGE, status.CanonicalCode()) << status;
 }
 
-
-TEST_F(EtcdConsistentStoreDeathTest, TestSetServingSTHChecksInconsistentSize) {
+TEST_F(EtcdConsistentStoreDeathTest,
+       TestSetServingSTHChecksInconsistentSizeDeathTest) {
```

Comment:
I'm pretty sure that the test case, rather than individual tests, are supposed to be called `*DeathTest`? The [How to Write a Death Test](https://code.google.com/p/googletest/wiki/V1_7_AdvancedGuide#How_to_Write_a_Death_Test) section says this:

> *Important*: We strongly recommend you to follow the convention of naming your test case (not test) `*DeathTest` when it contains a death test, as demonstrated in the above example. The [Death Tests And Threads](https://code.google.com/p/googletest/wiki/V1_7_AdvancedGuide#Death_Tests_And_Threads) section below explains why.

This change adds `DeathTest` to the tests (it was already there for the test cases), in direct opposition of this "important" recommendation. Maybe that's the thing to do, but I'm finding it suspicious that we're going directly against the recommendation here...

The FAQ later talks a bit more on this:

https://code.google.com/p/googletest/wiki/FAQ#Why_does_Google_Test_require_the_entire_test_case,_instead_of_in